app/src/forms/file/storage/objectStorageService.js

```diff
@@ -103,19 +108,12 @@ class ObjectStorageService {
         params.ContentType = mime.contentType(path.extname(fileStorage.originalName));
       }
 
-      return new Promise((resolve, reject) => {
-        // eslint-disable-next-line no-unused-vars
-        this._s3.upload(params, (err, data) => {
-          if (err) {
-            reject(err);
-          } else {
-            resolve({
-              path: data.Key,
-              storage: StorageTypes.OBJECT_STORAGE,
-            });
-          }
-        });
-      });
+      await this._s3.send(new PutObjectCommand(params));
+
+      return {
+        path: params.Key,
```

Comment:
The `PutObjectCommand` doesn't return the `Key` the way `upload` did, but the storage shouldn't be changing the key on us!